docs/docs/dev/explore/dpp-on-universal.md

```diff
@@ -66,4 +66,87 @@ For this to work, we must update our application to consume `redis` on `127.0.0.
 
 ::: tip
 You can parametrize your `Dataplane` definition, so you can reuse the same file for many `kuma-dp` instances or even services.
-:::
\ No newline at end of file
+:::
+
+## Lifecycle 
+
+On Universal there are 2 ways how Dataplane resource lifecycle could be managed: [Direct](#direct) and [Indirect](#indirect). 
+
+### Direct
+
+This is a recommended way to operate with Dataplane resources on Universal.
+
+#### Creation
+
+Pass Dataplane resource directly to `kuma-dp run` command. Dataplane resource could be a [Mustache template](http://mustache.github.io/mustache.5.html) in this case:
+
+_backend-dp-tmpl.yaml_
+```yaml
+type: Dataplane
+mesh: default
+name: {{ name }}
+networking:
+  address: {{ address }}
+  inbound:
+    - port: 8000
+      servicePort: 80
+      tags:
+        kuma.io/service: backend
+        kuma.io/protocol: http
+```
+
+The command with template parameters will look like this:
+```shell
+kuma-dp run \
+  --dataplane-file=backend-dp-tmpl.yaml \
+  --dataplane-var name=my-backend-dp \
+  --dataplane-var address=192.168.0.2 \
+  ...
+```
+
+When xDS connection between proxy and kuma-cp is established, Dataplane resource will be created automatically by kuma-cp. 
+
+#### Deletion
+
+If data plane proxy is shutdown gracefully, then Dataplane resource will be automatically deleted by kuma-cp. 
+
+If data plane proxy went down ungracefully, then Dataplane resource won't be deleted immediately. The following sequence 
+of the events should happen:
+1. Since xDS connection between proxy and kuma-cp is inactive, Dataplane proxy will be marked as Offline after `KUMA_METRICS_DATAPLANE_IDLE_TIMEOUT`, 
+by default it happens after 5 minutes.
+2. Offline data plane proxies are deleted automatically after `KUMA_RUNTIME_UNIVERSAL_DATAPLANE_CLEANUP_AGE`, by default it's 72h.
+
+It's guaranteed that Dataplane resource will be eventually cleaned up in case of **ungraceful** shutdown. 
+
+### Indirect
+
+The lifecycle is called "Indirect", because there is no strict dependency between Dataplane resource creation and the 
+startup of the data plane proxy. That's an old but still valid way to manage Dataplane resources. 
```

Comment:
I don't think it should qualified as old.

I think we should just say that this works if you have something that manages lifecycle. For example you'd do this with `Nomad`